jdaviz/configs/specviz/helper.py

```diff
@@ -144,7 +144,7 @@ def _set_scale(self, scale, axis, min_val=None, max_val=None):
             elif isinstance(min_val, u.Quantity):
                 # Convert user's value to axis' units
                 min_val = min_val.to(axis.unit).value
-            # If auto, set to min_valimum wavelength value
+            # If auto, set to min axis wavelength value
```

Comment:
This is a significant change. I couldn't figure out what a `valimum` was and decided that it was a corruption of minimum/maximum. I've tried to adjust these comments to match what the code appears to be doing.